src/main/java/net/countercraft/movecraft/combat/features/directors/AADirectors.java

```diff
@@ -97,14 +98,15 @@ private void processFireball(@NotNull SmallFireball fireball) {
         double speed = fireballVector.length() ; // store the speed to add it back in later, since all the values we will be using are "normalized", IE: have a speed of 1
         fireballVector = fireballVector.normalize(); // you normalize it for comparison with the new direction to see if we are trying to steer too far
 
-        Block targetBlock = DirectorUtils.getDirectorBlock(p, AADirectorRange);
-        Vector targetVector;
+        // the player is looking at nothing, shoot in that general direction
+        Vector targetVector = p.getLocation().getDirection();
```

Comment:
Same here

Reply:
Because DirectorUtils will ALWAYS execute a raycast, regardless of what you give it, sure the relevant code here could be included there however.